src/model/docker.ts

```diff
@@ -18,31 +19,7 @@ class Docker {
   }
 
   static async run(image, parameters, silent = false) {
-    const {
-      version,
-      workspace,
-      unitySerial,
-      runnerTempPath,
-      platform,
-      projectPath,
-      buildName,
-      buildPath,
-      buildFile,
-      buildMethod,
-      buildVersion,
-      androidVersionCode,
-      androidKeystoreName,
-      androidKeystoreBase64,
-      androidKeystorePass,
-      androidKeyaliasName,
-      androidKeyaliasPass,
-      androidTargetSdkVersion,
-      androidSdkManagerParameters,
-      customParameters,
-      sshAgent,
-      gitPrivateToken,
-      chownFilesTo,
-    } = parameters;
+    const { workspace, unitySerial, runnerTempPath, sshAgent } = parameters;
 
     const baseOsSpecificArguments = this.getBaseOsSpecificArguments(
```

Comment:
Should we also move this to the `ImageEnvironmentFactory` as well? (also fine in a followup)

Reply:
Issue exists to address